wprs

```
@@ -380,12 +380,36 @@ def run_remote_command(cmd: list[str], env: dict[str, str]) -> None:
   print(f'Executing remote command: {cmd!r}')
   subprocess.run(cmd, env=os.environ)
 
+def get_x_cursor_size() -> str:
+  default = '24'
+
+  if os.getenv('XCURSOR_SIZE'):
+    return os.getenv('XCURSOR_SIZE')
+
+  cmd = ['xrdb', '-query', '-get', 'Xcursor.size']
+  print(f'Executing local command: {cmd!r}')
+
+  # may not have x11-xserver-utils?
+  proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=sys.stderr, close_fds=True)
+  timeout_in_secs = 10
+  try:
+    (stdout, _) = proc.communicate(timeout=timeout_in_secs)
+    if stdout:
+      return stdout.replace('\n', '')
+  except subprocess.TimeoutExpired:
+    proc.kill()
+    (stdout, _) = proc.communicate()
+  except:
```

except Exception

A bare except catches more than you want usually, such as ctrl+c...